tests/test_submission.py

```diff
@@ -38,6 +38,75 @@ def test_list_submissions(db: Session, client: TestClient, logged_in_user):
     assert response.json()["results"][0]["id"] == str(submission.id)
 
 
+def test_get_metadata_submissions_mixs(db: Session, client: TestClient, logged_in_user):
+    # Create test submission
+    # submission1 has "Submitted- Pending Review" as the status (this is the one we want)
+    submission1 = fakes.MetadataSubmissionFactory(
+        status="Submitted- Pending Review",
+        metadata_submission={
+            "sampleData": {
+                "built_env_data": [
+                    {
+                        "samp_name": "Sample A",
+                        "env_medium": "Medium A",
+                        "env_broad_scale": "Broad Scale A",
+                        "env_local_scale": "Local Scale A",
+                    },
+                    {
+                        "samp_name": "Sample B",
+                        "env_medium": "Medium B",
+                        "env_broad_scale": "Broad Scale B",
+                        "env_local_scale": "Local Scale B",
+                    },
+                ]
+            },
+            "packageName": "Env Pkg 1",
+        },
+    )
+    db.commit()
+    response = client.request(method="get", url="/api/metadata_submission/mixs_report")
+    assert response.status_code == 200
+
+    # Check that the reponse payload is a TSV and that the result has the correct
+    # number of rows and information populated. The result should have 4 rows
```

Comment:
After having seen the test submission above having two biosamples in it, I expected this number to be 3 (i.e. 2 data rows + 1 header row = 3 total rows) instead of 4.

```suggestion
    # number of rows and information populated. The result should have 3 rows
```